doc/classes/ProjectSettings.xml

```diff
@@ -2673,6 +2673,7 @@
 		<member name="rendering/limits/spatial_indexer/update_iterations_per_frame" type="int" setter="" getter="" default="10">
 		</member>
 		<member name="rendering/limits/time/time_rollover_secs" type="float" setter="" getter="" default="3600">
+			Time (in seconds) before the global shader uniform [code]TIME[/code] resets to [code]0.0[/code].
```

Comment:
Use the full description from the other PR, as it's much more descriptive (and explains *why* this rollover is in place):

```suggestion
			When the [code]TIME[/code] shader built-in variable is incremented, it will roll over to [code]0.0[/code] when the value specified in [member rendering/limits/time/time_rollover_secs] is exceeded (in seconds). Since large floating-point values are less precise than small floating-point values, this should be set as low as possible to maximize the precision of the [code]TIME[/code] built-in variable in shaders. However, if this is set too low, shader animations may appear to restart from the beginning while the project is running.
```

Reply:
Most settings in ProjectSettings start with a terse summary of the setting itself, as either a noun ("The number of...") or a verb ("Sets...", "Overrides..."), and do not mention their own names. So in this case, we could start with something like this, before the full description:
> Maximum time (in seconds) before the `TIME` shader built-in variable rolls over to 0.0.

Also, this is my first PR on the main repo, so I'm not sure how I should be making iterative suggestions. Should I commit your suggested changes first before making my own? I checked [this page](https://docs.godotengine.org/en/latest/contributing/workflow/pr_review_guidelines.html#code-review-and-testing) but didn't see a clear answer.

Reply:
> Also, this is my first PR on the main repo, so I'm not sure how I should be making iterative suggestions. Should I commit your suggested changes first before making my own?

You should amend or rebase your branch, then force push your changes. See https://docs.godotengine.org/en/latest/contributing/workflow/pr_workflow.html#modifying-a-pull-request

Reply:
Thank you, I think I figured it out!